app/src/WalletProvider.tsx

```diff
@@ -116,6 +119,9 @@ function WalletProvider(props: React.PropsWithChildren<Record<string, never>>) {
               setNfts(data.assets);
```

Comment:
Also make sure to have `setError(undefined)` here too to reset the error when loading does work